README.md

```diff
@@ -11,21 +11,24 @@ This repository allows you to spin up a local test environment for the ``sdx-``
  - python3
 
 ### Getting started
+Make a directory called `sdx` e.g. `/users/[your_name]/sdx`.
 
-Export a ``SDX_HOME`` environment variable. This should point at a folder into
-which you wish to clone the rest of the services (e.g. ``/home/my-user/sdx``)
+This will contain all the files for the sdx services,
+
+Export an environment variable called ``SDX_HOME`` to the `sdx` folder.
 [see [configuration](#configuration)]
 
+Open your .bash_profile and add the line: ``export SDX_HOME=$HOME/[LOCATION OF SDX FOLDER]``
```

Comment:
This line and the other changed line (line 18) needs to change.  The line 18 line doesn't make any sense.  This line isn't accurate either.  Assuming you follow the instructions you'd have SDX_HOME be `$HOME/users/name/sdx` which doesn't exist.

Just change these lines to:
Open your .bash_profile and add the line: `export SDX_HOME=[LOCATION OF SDX FOLDER]`
Then apply the changes to the bash_profile to your session by doing `. ~/.bash_profile`